tutorials/evoked/plot_evoked_measures.py

```diff
@@ -0,0 +1,153 @@
+"""
+.. _tut_erp:
+
+Event Related Potentials (ERPs) Measures
+==================================================
+
+.. contents:: Here we cover the specifics of extracting data from ERPs, namely:
+   :local:
+   :depth: 1
+
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+###############################################################################
+# Create evoked from raw data
+
+# Read in raw data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+# Select only the the EEG channels
+raw.pick_types(meg=False, eeg=True, eog=True).load_data()
+
+# Create epochs from events, and reject epochs
+events = events = mne.read_events(event_fname)
+event_id = {'auditory/left': 1,
+            'auditory/right': 2,
+            'visual/left': 3,
+            'visual/right': 4}
+tmin, tmax = -0.2, 0.5
+reject = dict(eeg=180e-6, eog=150e-6)
+epochs_params = dict(events=events, event_id=event_id, tmin=tmin, tmax=tmax,
+                     reject=reject)
+epochs = mne.Epochs(raw, **epochs_params)
```

Comment:
I think you should pass the parameters to Epochs directly, since you're not re-using them anyway

Reply:
I ended up dropping this completely and reading in the evoked objet from the sample data.